ddtrace/internal/remoteconfig/__init__.py

```diff
@@ -1,19 +1,44 @@
+from typing import Optional
+
+from ddtrace.internal import agent
 from ddtrace.internal import atexit
 from ddtrace.internal import forksafe
 from ddtrace.internal.logger import get_logger
+from ddtrace.internal.remoteconfig.constants import REMOTE_CONFIG_AGENT_ENDPOINT
 from ddtrace.internal.remoteconfig.worker import RemoteConfigWorker
 
 
 log = get_logger(__name__)
 
 
+class _RemoteConfigException(Exception):
+    pass
+
+
 class RemoteConfig(object):
     _worker = None
     _worker_lock = forksafe.Lock()
 
+    @classmethod
+    def _check_remote_config_enable_in_agent(cls):
+        # type: () -> Optional[bool]
+        data = agent._healthcheck()
+        if data and data.get("endpoints"):
+            if REMOTE_CONFIG_AGENT_ENDPOINT in data.get("endpoints"):
+                return True
+        raise _RemoteConfigException(
```

Comment:
since we would have to handle this exception at some point, perhaps it's best to just log the error here and return `False` instead?

Reply:
:+1: